followed-accounts.js

```diff
@@ -0,0 +1,71 @@
+import { db } from './dbInstance.js'
+
+class FollowedActorsList extends HTMLElement {
+  connectedCallback () {
+    this.renderFollowedActors()
+  }
+
+  async renderFollowedActors () {
+    const followedActors = await db.getFollowedActors()
+    this.innerHTML = followedActors
```

Comment:
Can't use innerHTML for this. Loop over and appnd divs to be safe combined with `.innerText`. Else people can inject scripts in the `url` property and other nasty stuff

Reply:
Ah, my bad.